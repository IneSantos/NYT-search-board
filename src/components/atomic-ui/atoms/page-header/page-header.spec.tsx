import React from "react";
import { render } from '@testing-library/react';
import PageHeader from "./page-header";

describe("PageHeader atom tests", () => {
  test(`PageHeader component render`, () => {
    const { container } = render(
      <PageHeader/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
