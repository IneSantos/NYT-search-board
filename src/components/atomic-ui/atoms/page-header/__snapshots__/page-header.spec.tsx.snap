// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader atom tests PageHeader component render 1`] = `
<header>
  <div
    class="banner"
  >
    “The New York Times” article search application
  </div>
</header>
`;
